src/Plugin.InAppBilling/InAppBilling.android.cs

```diff
@@ -316,7 +316,7 @@ async Task<InAppBillingPurchase> UpgradePurchasedSubscriptionInternalAsync(strin
         /// <param name="obfuscatedAccountId">Specifies an optional obfuscated string that is uniquely associated with the user's account in your app.</param>
         /// <param name="obfuscatedProfileId">Specifies an optional obfuscated string that is uniquely associated with the user's profile in your app.</param>
         /// <returns></returns>
-        public async override Task<InAppBillingPurchase> PurchaseAsync(string productId, ItemType itemType, string obfuscatedAccountId = null, string obfuscatedProfileId = null, string subOfferToken = null, CancellationToken cancellationToken = default)
+        public async override Task<InAppBillingPurchase> PurchaseAsync(string productId, ItemType itemType, string obfuscatedAccountId = null, string obfuscatedProfileId = null, string subOfferToken = null, bool shouldConsoleLog = false, CancellationToken cancellationToken = default)
```

Comment:
The interface changed so there are unrelated platforms that are being updated to match.